hasura.planx.uk/metadata/tables.yaml

```diff
@@ -206,6 +206,8 @@
                 _is_null: true
             - deleted_at:
                 _is_null: true
+            - email:
+                _neq: ""
```

Comment:
Perfect 👌 

So that the full behaviour of this table and associated permissions are documented in one place, can we add a very simple test in `hasura.planx.uk/tests/lowcal_sessions.test.js` that verifies that the Public user cannot access an session with an email of `""`?